app/Controllers/Approval.php

```diff
@@ -196,7 +196,7 @@ public function reject(int $userId, int $eventId): ResponseInterface
         if ($canBeRejected instanceof ResponseInterface) {
             return $canBeRejected;
         }
-        if (!$canBeRejected) {
+        if ($canBeRejected === false) {
```

Comment:
How can that make a difference? 😅

Reply:
As long as `$canBeRejected` is a `bool`, this won't make a difference. But look at this:
```php
$s = ''; // empty string

if (!$s) {
	echo 'Web languages were a mistake.'; // <- will be printed!
}

if ($s === false) {
	echo 'This won\'t get printed.';
}
```
I guess this is similar to what JavaScript would do. This is just stupid!